Snakefile

```diff
@@ -1402,6 +1426,92 @@ rule generate_methods_differential_rnaseq:
         cp {output.methods} {wildcards.accession}-analysis-methods.tsv
         """
 
+rule deconvolution:
+    """
+    Runs three deconvolution tools (DWLS, FARDEEP, EpiDIS) for experiments selected in deconvolution.yaml.
+    """
+    conda: "envs/deconvolution.yaml"
+    log: "logs/{accession}-deconvolution.log"
+    #resources: mem_mb=get_mem_mb
+    resources: mem_mb=64000
+    threads: 16
+    input: 
+        fpkms="{accession}-fpkms.tsv.undecorated",
+	methods=get_methods_file,
+	#methods=rules.generate_methods_baseline_rnaseq.output,
+        sdrf=get_sdrf()
+    params:
+        signature_dir=config["deconv_ref"] + get_organism()
+    output:
+        proportions="{accession}-deconvolution.proportions.tsv",
+	methods="{accession}-analysis-methods.tsv",
```

Comment:
Is this output having the same name than other output generated in other rules? e.g. `rule generate_methods_baseline_rnaseq`

So, you need to clarify. Are you modifying an existing file created by other rule, or perhaps the file here (if new) should be named `{accession}-deconv-methods.tsv`?

Reply:
@pmb59 I am appending the output file of `rule generate_methods_differential/baseline_rnaseq`. I am not sure wether to rename the file or not. I thought it would be problematic if the deconvolution methods file has a different name than for the other experiments ...